### PR TITLE
Update Team kernels to single policies

### PR DESCRIPTION
**Pull request by artv3**
Removes unused policies in RAJA Teams kernels. Resolves https://github.com/LLNL/RAJAPerf/issues/204

### CODE REVIEW

File: src/apps/DIFFUSION3DPA-Cuda.cpp [hunk 1]
```diff
@@ -151,26 +151,24 @@ void DIFFUSION3DPA::runCudaVariant(VariantID vid) {
     DIFFUSION3DPA_DATA_SETUP_CUDA;
 
     using launch_policy =
-        RAJA::expt::LaunchPolicy<RAJA::expt::seq_launch_t,
-                                 RAJA::expt::cuda_launch_t<true>>;
+        RAJA::expt::LaunchPolicy<RAJA::expt::cuda_launch_t<true>>;
```
Comment (rhornung67) [inline src/apps/DIFFUSION3DPA-Cuda.cpp:154]: Is the bool template arg used to indicate async launch? Please use a local variable with a descriptive name; e.g., constexpr bool async = true; and pass that so the code is clear.

File: src/apps/DIFFUSION3DPA-Hip.cpp [hunk 2]
```diff
@@ -152,26 +152,24 @@ void DIFFUSION3DPA::runHipVariant(VariantID vid) {
     DIFFUSION3DPA_DATA_SETUP_HIP;
 
     using launch_policy =
-        RAJA::expt::LaunchPolicy<RAJA::expt::seq_launch_t,
-                                 RAJA::expt::hip_launch_t<true>>;
+        RAJA::expt::LaunchPolicy<RAJA::expt::hip_launch_t<true>>;
```
Comment (rhornung67) [inline src/apps/DIFFUSION3DPA-Hip.cpp:155]: same comment about local var for template arg here and elsewhere

**Review by rhornung67 — CHANGES_REQUESTED** [review]
Please see my comments.

File: src/apps/DIFFUSION3DPA-Cuda.cpp [hunk 3]
```diff
@@ -150,27 +150,27 @@ void DIFFUSION3DPA::runCudaVariant(VariantID vid) {
 
     DIFFUSION3DPA_DATA_SETUP_CUDA;
 
+    constexpr bool async = true;
```
Comment (rhornung67) [inline src/apps/DIFFUSION3DPA-Cuda.cpp:153]: 👍

**Review by MrBurmark — APPROVED** [review]

**Pull request merged**